.github/workflows/e2e.yml

```diff
@@ -0,0 +1,80 @@
+name: Integration Testing
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: cs9 x86_64 e2e
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTS_DIR: exported-artifacts
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y install podman
+
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          # Use githash of a tested commit instead of merge commit
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: rpm-artifacts
+          path: ${{ env.ARTIFACTS_DIR}}
+
+      - name: build hirte tests containers
+        run: |
+          podman build -f tests/Containerfile -t hirte-image .
+
+      - name: run hirte containers
+        run: |
+          podman run -d --net podman --rm -p 2020:2020 --name hirte-mgr \
+          $(podman images *hirte-image --format {{.ID}})
+          podman run -d --net podman --rm --name hirte-node-bar \
+          $(podman images *hirte-image --format {{.ID}})
+          podman run -d --net podman --rm --name hirte-node-foo \
+          $(podman images *hirte-image --format {{.ID}})
+          PODMAN_NET=$(hostname -I | awk '{print $1}')
+          echo "PODMAN_NET=$PODMAN_NET" >> $GITHUB_ENV
+
+      - name: run hirte orchestrator
+        run: |
+          podman exec -it hirte-mgr bash -c "man hirte.conf | \
+               grep -A 5 \"\[hirte\]\" | sed -e 's/^ *//g' \
+               -e 's/journald/stderr/' \
+               -e 's/agent.*/foo,bar/' > /etc/hirte/hirte.conf"
```

Comment:
I'd really prefer to clearly define what's changed here. So instead of relying on man page content, I'd update specific options, for example

```
# Allow nodes foo and bar to connect
          podman exec -it hirte-mgr \
              sed -i "s/\(AllowedNodeNames=\)\(.*\)/\1foo,bar/" /etc/hirte/hirte.conf
          # Change LogLevel to DEBUG
          podman exec -it hirte-mgr \
              sh -c 'echo "LogLevel=DEBUG" >> /etc/hirte/hirte.conf'
```

Reply:
It's not empty, it should contain https://github.com/containers/hirte/blob/main/config/hirte/hirte.conf after hirte RPM is installed

Reply:
Ack